app/assets/stylesheets/member-facing/registration.scss

```diff
@@ -8,8 +8,29 @@
     margin-bottom: 20px;
   }
 
+  p {
+    padding-bottom: 10px;
```

Comment:
Yeah, assigning styles to tags, and to things dependent on nesting isn't seen as ideal these days... it's not a big deal but using classes is better and it lets you name your elements more explicitly.